apps/meteor/app/lib/server/functions/saveUser.ts

```diff
@@ -409,10 +407,9 @@ export const saveUser = async function (userId, userData) {
 		sendPassword = false;
 	}
 
-	const updateUser = {
-		$set: {},
-		$unset: {},
-	};
+	const updateUser: DeepWritable<UpdateFilter<IUser>> = {};
```

Comment:
If you're doing the type DeepWritable, can't you just define $set/$unset inside here? What error is ts giving? 👀

Reply:
It says that $set and $unset are possibly undefined. If we'd like to assign these properties here, we'd have to change the typing so as to make TS sure that they've been defined